### PR TITLE
Add endpoint for retrieving counts of aws account resources

### PR DESCRIPTION
**Pull request by kralle333**
closes dfds/cloudplatform/issues/1923

## Additional Review Notes
Also added more db seed data so that finout code is still displayable after pending-deletion-xxx has been deleted.

### CODE REVIEW

File: src/SelfService/Application/PlatformDataApiRequesterService.cs [hunk 1]
```diff
@@ -126,16 +192,51 @@ public async Task<MyCapabilityCosts> GetMyCapabilitiesCosts(UserId userId, int d
...
+        List<CapabilityAwsResourceCounts> resourceCounts = new List<CapabilityAwsResourceCounts>();
+        foreach (var accountsCount in awsResourceCounts)
+        {
+            if (!awsAccountIdToCapabilityIdMap.TryGetValue(accountsCount.AwsAccountId, out var capabilityId))
+            {
+                // not an error, remember my capabilities are a subset of all capabilities
+                continue;
+            }
+
+            if (!mappedCounts.TryGetValue(accountsCount.AwsAccountId, out var counts))
```
Comment (andreasfrisch) [inline src/SelfService/Application/PlatformDataApiRequesterService.cs:222]: How does this even happen?
They are based on the same source data, are they not?

**Review by andreasfrisch — APPROVED** [review]
It looks good to me.
I left some comments, one of them a question, but they are not blocking.

One note though; It feels as if this PR is a mix between introducing aws resource counts and refactoring the code around costs.
In a perfect world (my version of 'perfect') those two things are separate PRs

File: src/SelfService/Application/PlatformDataApiRequesterService.cs [hunk 2]
```diff
@@ -107,15 +156,32 @@ HttpClient httpClient
...
 
-    public async Task<MyCapabilityCosts> GetMyCapabilitiesCosts(UserId userId, int daysWindow)
+    private Dictionary<string, List<PlatformDataApiAwsResourceCount>> ToAwsAccountMap(
+        List<PlatformDataApiAwsResourceCounts> counts
+    )
     {
-        var timeSeriesWithValidCapabilities = await FetchCapabilityCosts(
-            new KeyValuePair<string, string>[] { new(QueryParamDaysWindow, daysWindow.ToString()) }
-        );
+        Dictionary<string, List<PlatformDataApiAwsResourceCount>> costs =
```
Comment (andreasfrisch) [inline src/SelfService/Application/PlatformDataApiRequesterService.cs:163]: Is it really costs?

File: src/SelfService/Application/PlatformDataApiRequesterService.cs (hunk 1 above)
Comment (andreasfrisch) [reply src/SelfService/Application/PlatformDataApiRequesterService.cs:222]: Discussed on slack. This is fine.

**Review by andreasfrisch — APPROVED** [review]
Still looks good to me :shipit:

File: src/SelfService/Application/PlatformDataApiRequesterService.cs (hunk 2 above)
Comment (kralle333) [reply src/SelfService/Application/PlatformDataApiRequesterService.cs]: nope... renamed :)

**Pull request merged**